pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1069,6 +1067,7 @@ BPF_HASH(uts_ns_filter, string_filter_t, eq_t, 256);               // filter eve
 BPF_HASH(comm_filter, string_filter_t, eq_t, 256);                 // filter events by command name
 BPF_HASH(cgroup_id_filter, u32, eq_t, 256);                        // filter events by cgroup id
 BPF_HASH(binary_filter, binary_t, eq_t, 256);                      // filter events by binary path and mount namespace
+BPF_HASH(event_scopes_map, u32, u64, MAX_EVENT_ID);
```

Comment:
Please change it to events_map. In the future, we'll have the value as a compound type with not only scopes value.